src/org/usfirst/frc/team2706/robot/OI.java

```diff
@@ -48,22 +52,28 @@ public OI(Joystick driverStick, Joystick controlStick) {
         // Joystick for driving the robot around
         this.driverStick = driverStick;
 
-        // TODO we need to start using controlStick and not driverStick for non-testing buttons
-
-        EJoystickButton backLeftButton = new EJoystickButton(driverStick, 5);
-        backLeftButton.runWhileHeld(new AlignAndDistance(24));
-
         EJoystickButton a = new EJoystickButton(driverStick, 1);
-        a.runWhileHeld(new Climb());
+        a.runWhileHeld(new ClimbManually());
 
         EJoystickButton b = new EJoystickButton(driverStick, 2);
         b.whenPressed(new GearHandlerToggle());
-
+        
+        EJoystickButton c = new EJoystickButton(driverStick, 3);
+        c.toggleWhenPressed(climbAutomatically);
+
+        // test the camera integration
+        EJoystickButton cameraButton = new EJoystickButton(driverStick, 3);
```

Comment:
"Maybe tell us either in the name or the comment what button 3 is on the joystick?" This might have got buried when I pushed formatting changes.

Also, you are using the same button for the same button as `climbAutomatically`.